titan-server/src/accounts/routes.rs

```diff
@@ -7,7 +7,11 @@ use serde::{Deserialize, Serialize};
 use crate::accounts::users;
 use crate::accounts::file_entries;
 use crate::accounts::file_entry_types;
+<<<<<<< 18d37b5084a80e3cf2ec0eae998834213c5d9e3d
```

Comment:
It looks like a merge conflict snuck in during a rebase?

Reply:
Yeah, it's fixed now.